vendor/github.com/kubernetes-csi/external-snapshotter/client/v3/clientset/versioned/typed/volumesnapshot/v1/volumesnapshotcontent.go

```diff
@@ -0,0 +1,184 @@
+/*
```

Comment:
I am having a hard time to understand these generated files, taking:
clientset/versioned/typed/volumesnapshot/v1/volumesnapshotcontent.go as an example, there seems to be two identical ones generated:
vendor/github.com/kubernetes-csi/external-snapshotter/client/v3/clientset/versioned/typed/volumesnapshot/v1/volumesnapshotcontent.go
and 
client/clientset/versioned/typed/volumesnapshot/v1/volumesnapshotcontent.go

could you shed some light what are them for?

Reply:
Actually only `client/clientset/versioned/typed/volumesnapshot/v1/volumesnapshotcontent.go` is generated by code-generator. `vendor/github.com/kubernetes-csi/external-snapshotter/client/v3/clientset/versioned/typed/volumesnapshot/v1/volumesnapshotcontent.go` is generated by running "go mod vendor" because we have a external-snapshotter/vendor folder that already has those files for v1beta1.